code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -27,29 +27,34 @@
 	speak_emote = list("chitters")
 	emote_hear = list("chitters")
 	speak_chance = 5
+	speed = 1
 	turns_per_move = 5
 	see_in_dark = 10
 	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab/spider = 2, /obj/item/reagent_containers/food/snacks/spiderleg = 8)
 	response_help  = "pets"
 	response_disarm = "gently pushes aside"
 	response_harm   = "hits"
-	maxHealth = 200
-	health = 200
-	obj_damage = 60
-	melee_damage = 15
+	initial_language_holder = /datum/language_holder/spider // Speaks buzzwords, understands buzzwords and common
+	maxHealth = 100
+	health = 100
+	obj_damage = 25
+	melee_damage = 25
+	poison_per_bite = 0
```

Comment:
I intentionally had the "brown" spiders be stronger and not have venom, so that they're differentiated from the hunter/viper spiders. That way it encourages working together a bit more, with the hunters being able to paralyze but not quite having the stopping power of the slower and tankier brown spiders. I'll probably have to tweak damage for both of these more to get to that point, though.